pyof/v0x01/asynchronous/packet_in.py

```diff
@@ -64,4 +64,4 @@ def __init__(self, xid=None, buffer_id=NO_BUFFER, total_len=None,
         self.total_len = total_len
         self.in_port = in_port
         self.reason = reason
-        self.data = data
+        self.data = BinaryData() if data is None else data
```

Comment:
`BinaryData()` -> `b''`; it's bad having GenericTypes when they are not needed.
The cleaner way is to set `b''` directly in the `__init__` method - same behavior, better readability.